docs/setup/yarn_setup.md

```diff
@@ -100,11 +100,13 @@ Usage:
      -D <arg>                        Dynamic properties
      -d,--detached                   Start detached
      -jm,--jobManagerMemory <arg>    Memory for JobManager Container [in MB]
+     -nm,--name                      Set a custom name for the application on YARN
```

Comment:
This short option `-nm` can be very easily confused with the `-n` parameter which sets the number of containers. I would remove the short option and just have the long one `--name`.